src/uct/ib/base/ib_iface.h

```diff
@@ -145,6 +148,7 @@ struct uct_ib_iface {
         uint8_t             sl;
         uint8_t             traffic_class;
         size_t              max_iov;             /* Maximum buffers in IOV array */
+        int                 res_domain;          /* Enable multiple resource domains */
```

Comment:
can move this var before max_iov to reduce overall struct size

Reply:
moved